### ISSUE TITLE
GridFieldEditableColumns() not saving/creating

### ISSUE THREAD

**Issue by n8-sd** [symbiote/silverstripe-gridfieldextensions#147]
No way to save, can't save new either.

Is it me?

```
  $fields->addFieldToTab('Root.Main', GridField::create(
             'Categories',
            'News Categories',
            $this->Categories(),
            GridFieldConfig::create()
                ->addComponent($add = new GridFieldAddNewInlineButton('toolbar-header-right'))
                ->addComponent(new GridFieldToolbarHeader())
                ->addComponent(new GridFieldTitleHeader())
                ->addComponent(new GridFieldEditableColumns())
                ->addComponent(new GridFieldDeleteAction())
                ->addComponent(new GridFieldOrderableRows('SortOrder'))
        ));
        $add->setTitle('Add a Category');
```

**Comment by jonom** [comment]
If you hit the page's Save or Save & Publish button see if it works then. I don't find it intuitive but that seems to work for me.

**Comment by robbieaverill** [comment]
Going by #148 I assume this is a UX problem, which can be addressed in #148. If this is another problem please feel free to reopen.
